content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -101,6 +101,9 @@ effect `PreferNoSchedule` then Kubernetes will *try* to not schedule the pod ont
 * if there is at least one un-ignored taint with effect `NoExecute` then the pod will be evicted from
 the node (if it is already running on the node), and will not be
 scheduled onto the node (if it is not yet running on the node).
+* if there is only one tolerated taint with effect `NoExecute` then the pod will still be scheduled onto the node even if
```

Comment:
This makes very little sense, sorry.

I think this should be pointed out as a note to the following paragraphs, which describe the `tolerationSeconds` functionality. There is a toleration anyways, so the third bullet applies, thus the pod can be scheduled.

Reply:
@zhouya0 I'm afraid I agree with @alculquicondor 

Was there a part of the page you found confusing? Maybe knowing that will help someone propose a different change that still reduces the confusion you noticed.

Reply:
Context: kubernetes/kubernetes#92170

Reply:
More specifically, https://github.com/kubernetes/kubernetes/issues/92170#issuecomment-646352425

Reply:
@alculquicondor @sftim 
I followed your suggestions. Please review this again :)